### PR TITLE
clasa noua

### PR DESCRIPTION
**Pull request by netcupeter**
Verificati

### CODE REVIEW

File: netcupeter/src/main/java/com/netcu/Hello.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.netcu;
+
+/**
+ * Created by netcu on 6/3/2017.
+ */
+public class Hello {
+    public static void main(String[] args) {
+        System.out.println("Hello Netcu");
```
Comment (eugenbarbu) [inline netcupeter/src/main/java/com/netcu/Hello.java]: Salut

File: netcupeter/src/main/java/com/netcu/Hello.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package com.netcu;
+
+/**
+ * Created by netcu on 6/3/2017.
+ */
+public class Hello {
```
Comment (tanaseanca90) [inline netcupeter/src/main/java/com/netcu/Hello.java]: ce face clasa?

File: netcupeter/src/main/java/com/netcu/Hello.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package com.netcu;
+
+/**
+ * Created by netcu on 6/3/2017.
```
Comment (silviudeaconu) [inline netcupeter/src/main/java/com/netcu/Hello.java]: da, e bine!

**Comment by netcupeter** [comment]
ok!

**Comment by seropian** [comment]
looks good, please merge

**Pull request merged**